src/App.tsx

```diff
@@ -1,9 +1,87 @@
+import { Route, Routes, useLocation } from 'react-router-dom';
 import './App.scss';
+import { useEffect } from 'react';
+// import { Product } from './Components/ProductCards/Product';
+import { FavoriteContextProvider } from './FavoriteContext';
+import { HomePAge } from './Components/Pages/HomePage/HomePage';
+import { Header } from './Components/Header/Header';
```

Comment:
1. rename `Components` to `components` (from lower case)
create `index.ts` file to each component to export it like
```
components
--Pages
----HomePage
------HomePage.tsx
------HomePage.scss
------index.ts   <--------- export * from './HomePage'
```
and add general `index.ts` file directly to the `components` folder
and do like

```
export * from './Header'
export * from './Pages/HomePage'
```
etc

and you can import all necessary components from 1 import

```
import { HomePage, Header, NotFoundPage } from './components'
```